nmdc_runtime/api/endpoints/nmdcschema.py

```diff
@@ -35,6 +36,41 @@ def get_nmdc_schema_version():
     return version("nmdc_schema")
 
 
+@router.get("/nmdcschema/collection_stats")
```

Comment:
This is part of PR https://github.com/microbiomedata/nmdc-runtime/pull/287, so I won't review it here.